ibm_catalog.json

```diff
@@ -0,0 +1,98 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-observability",
+        "label": "Deployment of IBM Cloud Observability Resources",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "reference_architecture",
+          "solution"
+        ],
+        "keywords": [
+          "slz",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "observability",
+          "observability instances",
+          "log analysis",
+          "cloud monitoring"
+        ],
+        "short_description": "Deploys the IBM Cloud Observability Resources",
+        "long_description": "Deploys the IBM Cloud Observability Resources - Log Analysis and Cloud Monitoring",
```

Comment:
Solutions which support configuring IBM Cloud Observability resources for logging, monitoring and activity tracking

Reply:
Did not add activity tracking as we are not creating/configuring AT.

Reply:
But we are adding Activity Tracking - we are creating a route and pushing AT events to COS. We are just not crating a (logdna based) Activity Tracker instance